### PR TITLE
Feature/AT-6300 As a developer, create header in SPA.

### PR DESCRIPTION
**Pull request by jGonzalez-CCPO**
• Unsaved changes
Create header component
Theme header component

Failed to load
ATAT Design Library: Figma 

Acceptance Criteria
Create header component
USA Gov header
Unclassified Header
ATAT header with blue background
Toggle menu 
Menu with icons appear after login

### CODE REVIEW

File: src/components/ATATHeaderNav.vue [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+      v-for="item in navData.items"
+      :class="item.cssClass"
+      :id="item.cssClass"
+      :title="item.title"
+      @click="itemCLicked(item)"
+      :key="item.id"
+    >
+      <span>{{ item.title }}</span>
+    </v-btn>
+
```
Comment (tonysumner-ccpo) [inline src/components/ATATHeaderNav.vue:14]: remove commented code below

File: src/components/ATATHeaderNav.vue [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  get navData(): Nav {
+    if (this.loginStatus) {
+      return all_navs["login"];
+    } else {
+      return all_navs["logout"];
+    }
+  }
+
+  private login(): void {
+    console.log("action login");
```
Comment (tonysumner-ccpo) [inline src/components/ATATHeaderNav.vue:106]: remove console.log

File: src/components/ATATHeaderNav.vue [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    this.$store.dispatch("login");
+  }
+  private logout(): void {
+    console.log("action logout");
```
Comment (tonysumner-ccpo) [inline src/components/ATATHeaderNav.vue:110]: remove console.log

File: src/views/Home.vue [hunk 4]
```diff
@@ -18,8 +18,9 @@
       <v-row>
         <v-col class="d-flex justify-center mb-9">
           <v-btn
-            to="/dashboard"
-            large
+            @click="login()"
+            elevation="2"
+            x-large
```
Comment (tonysumner-ccpo) [inline src/views/Home.vue:23]: - [ ] Change x-large to large

File: src/views/Home.vue [hunk 5]
```diff
@@ -56,5 +57,11 @@
 import Vue from "vue";
 import { Component } from "vue-property-decorator";
 @Component({})
-export default class Home extends Vue {}
+export default class Home extends Vue {
+  private login(): void {
+    console.log("action login");
```
Comment (tonysumner-ccpo) [inline src/views/Home.vue:62]: remvoe console.log

File: src/components/ATATHeaderNav.vue (hunk 1 above)
Comment (jGonzalez-CCPO) [reply src/components/ATATHeaderNav.vue]: done

File: src/components/ATATHeaderNav.vue (hunk 2 above)
Comment (jGonzalez-CCPO) [reply src/components/ATATHeaderNav.vue]: done

File: src/components/ATATHeaderNav.vue (hunk 3 above)
Comment (jGonzalez-CCPO) [reply src/components/ATATHeaderNav.vue]: done

File: src/views/Home.vue (hunk 4 above)
Comment (jGonzalez-CCPO) [reply src/views/Home.vue]: done

File: src/views/Home.vue (hunk 5 above)
Comment (jGonzalez-CCPO) [reply src/views/Home.vue]: done

**Review by DGabriel-CCPO — APPROVED** [review]

**Pull request merged**